docs/example-data.json

```diff
@@ -1065,7 +1068,8 @@
         "begin_time": 1584512636,
         "end_time": 1584512638,
         "weight": null,
-        "marked": false,
+        "speech_state": null,
+        "note": "",
```

Comment:
Shouldn't note be either `null` or not existent if `point_of_order` is false?

Reply:
`null` is better; otherwise the datavalidator complains about a missing field (We do want to have every field explicitly here even though `undefined===null`)